tests/nodeset-import/check_nodeset_importSpeed.c

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Whats the total time of this test? If it takes a serious amount of time the CI-times might inrease further. (20min for each travis build is already quite some time at the moment)

Reply:
should be below 10-20s